src/contentmanagerview.cpp

```diff
@@ -23,6 +23,9 @@ ContentManagerView::ContentManagerView(QWidget *parent)
         if (index.column() == (mp_ui->m_view->model()->columnCount() - 1))
             return;
 
+        /* The descriptions will only need one column to display. */
+        mp_ui->m_view->setFirstColumnSpanned(0, index, true);
+
```

Comment:
Please convert the handler lambda containing this change into a named slot of `ContentManagerView` (do that in a separate commit). Then modify it in a separate commit.